T/Thrift/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Thrift"
+version = v"0.16.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/apache/thrift.git", "2a93df80f27739ccabb5b885cb12a8dc7595ecdf")
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/thrift
+
+# Needed as https://github.com/apache/thrift/pull/2518 isn't released yet
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+
+mkdir ../build && cd ../build
```

Comment:
```suggestion
mkdir build && cd build
```

Reply:
There's already a `build` directory in the source folder.

Reply:
Give it another name then, it doesn't matter, but don't build in another directory under `/workspace/srcdir` otherwise license files won't be found automatically: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=21926&view=results